app/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>openmrs-owa-orderentry</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+  <!-- openmrs favicon -->
+  <link rel="shortcut icon" type="image/ico" href="../../images/openmrs-favicon.ico" />
+  <link rel="icon" type="image/png" href="../../images/openmrs-favicon.png" />
+
+  
+  <link rel="stylesheet" href="../../ms/uiframework/resource/coreapps/styles/patientHeader.css"
+    type="text/css" />
+  <link rel="stylesheet" href="../../ms/uiframework/resource/uicommons/styles/styleguide/index.css"
+    type="text/css" />
+  <link rel="stylesheet" href="../../ms/uiframework/resource/appui/styles/header.css" type="text/css"
+  />
+  <link rel="stylesheet" href="../../ms/uiframework/resource/uicommons/styles/styleguide/jquery-ui-1.9.2.custom.min.css" type="text/css"/>
```

Comment:
This is fine for a now (and I realize it's what we are doing in the Order Entry OWA as well), but long-term we don't want to have to rely on css files being pulled in here via the UI Framework.   It would be great for some ideas on the best way to bundle css files within an OWA... you can see a Talk thread about this here:

https://talk.openmrs.org/t/dry-and-all-these-new-reactjs-owas/17203

Of the two main points that Darius makes in his original post, the second one (not replicating the Header in every OWA) is something we solved by creating the openmrs-react-components project.  However, we never really addressed his first point... you see here that we require the coreapps, uicommons and appui modules to be running to pull in their style sheets.  We should come up with a better means of doing things... I'm interested in your thoughts about the best way to do it?  I know we've done some bundling with scss in the past.

In the sort-to-mid term we can keep this like it is so we can continue to move forward but if you guys have some thoughts on how to best structure our css going forward, maybe we can start creating some tickets for future work refactoring this (and the similar code in the orderentry OWA).